examples/bootstrap-example.html

```diff
@@ -62,9 +62,16 @@
            ng-class="{red: !itParses}" ng-model="schemaJson" class="form-control schema"></div>
     </div>
   </div>
+  <div>
+      <p>
+          <small>
+              <a href="../jslicense.html" rel="jslicense">JavaScript license information</a>
+          </small>
+      </p>
+  </div>
 </div>
-<script type="text/javascript" src="http://code.jquery.com/jquery-2.1.1.min.js"></script>
-<script type="text/javascript" src="http://cdn.jsdelivr.net/g/jquery.ui@1.10"></script>
+<script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
```

Comment:
Is there a reason you changed these to https? https can cause up to 5x slower load, and i personally don't see a reason that one would need to load these over https, its jquery.